pkgs/games/fallout-ce/build.nix

```diff
@@ -28,9 +29,11 @@ stdenv.mkDerivation (finalAttrs: {
 
   installPhase = ''
     runHook preInstall
-    install -D ${name} $out/bin/${name}
+
+    install -D ${name} $out/bin/${name}-unwrapped
     install -D ${./wrapper.sh} $out/bin/wrapper.sh
-    wrapProgram $out/bin/${name} --run $out/bin/wrapper.sh
+    wrapProgram $out/bin/${name}-unwrapped $out/bin/${name} --run $out/bin/wrapper.sh
```

Comment:
```suggestion
    makeWrapper $out/bin/${pname}-unwrapped $out/bin/${pname} --run $out/bin/wrapper.sh
```

Reply:
`wrapProgram` only takes one positional arg, I suspect you meant `makeWrapper` https://nixos.org/manual/nixpkgs/stable/#fun-makeWrapper